socorro/lib/migrations.py

```diff
@@ -11,17 +11,36 @@
 
 import os
 
-
-def load_stored_proc(op, filelist):
+def get_local_filepath(filename):
     """
-    Takes a list of files and the alembic op object as arguments
-    Load and run CREATE OR REPLACE function commands from files
+    Helper for finding our raw SQL files locally.
+
     Expects files to be in:
         $CWD/socorro/external/postgresql/raw_sql/procs/
     """
     app_path=os.getcwd()
+    return (app_path + '/socorro/external/postgresql/raw_sql/procs/' + filename)
+
+
+def local_file_exists(filename):
+    sqlfile = get_local_filepath(filename)
+    return os.path.isfile(sqlfile)
+
+
+def load_stored_proc(op, filelist):
+    """
+    Takes the alembic op object as arguments and a list of files as arguments
+    Load and run CREATE OR REPLACE function commands from files
+    """
     for filename in filelist:
-        sqlfile = app_path + '/socorro/external/postgresql/raw_sql/procs/' + filename
+        sqlfile = get_local_filepath(filename)
+        # Capturing "file not exists" here rather than allowing
+        # an exception to be thrown. Some of the rollback scripts
+        # would otherwise throw unhelpful exceptions when a SQL
+        # file is removed from the repo.
+        if local_file_exists(sqlfile) == False:
```

Comment:
Generally, doing `something == True` or `something == False` is considered an anti-pattern and the preferred pattern is to just do `if something:` or `if not something:`. That way it works with `0`, `None`, `False` or `[]` etc.